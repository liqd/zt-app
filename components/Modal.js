import React from 'react';
import { View } from 'react-native';
import { BottomSheet } from 'react-native-elements';
import { ModalItem } from './ModalItem';
import { styles } from './Modal.styles';

export const Modal = (props) => {
  return (
    <BottomSheet
      isVisible={props.isVisible}
      containerStyle={styles.backdrop}
    >
      <View style={styles.list}>
        {props.modalItems.map((modalItem, idx) =>
          <ModalItem key={idx} item={modalItem} />)}
      </View>
    </BottomSheet>
  );
};
