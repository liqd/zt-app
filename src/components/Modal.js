import React from 'react'
import { View } from 'react-native'
import { BottomSheet } from '@rneui/themed'

import { styles } from './Modal.styles'
import { ModalItem } from './ModalItem'

export const Modal = (props) => {
  return (
    <BottomSheet
      isVisible={props.isVisible}
      containerStyle={styles.backdrop}
    >
      <View style={styles.list}>
        {props.modalItems.map((modalItem, idx) =>
          <ModalItem key={idx} item={modalItem} />)}
      </View>
    </BottomSheet>
  )
}
