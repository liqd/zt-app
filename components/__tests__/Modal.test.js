import React from 'react'
import { render } from '@testing-library/react-native'

import { Modal } from '../Modal'

test('Test Modal isVisible false', () => {
  const { toJSON} = render(<Modal isVisible={false} modalItems={[]} />)
  expect(toJSON()).toMatchSnapshot()
})

test('Test Modal isVisible true', () => {
  const { toJSON} = render(<Modal isVisible={true} modalItems={[]} />)
  expect(toJSON()).toMatchSnapshot()
})

test('Test Modal with ModalItems', () => {
  const modalItems = [
    {isCancel: false, isText: false},
    {isCancel: false, isText: false, action: () => {}},
    {isCancel: true, isText: false},
    {isCancel: false, isText: true},
    {isCancel: true, isText: true},
  ]
  const { toJSON} = render(<Modal isVisible={true} modalItems={modalItems} />)
  expect(toJSON()).toMatchSnapshot()
})
