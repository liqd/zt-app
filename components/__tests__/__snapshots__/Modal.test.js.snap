// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Modal isVisible false 1`] = `
<Modal
  animationType="slide"
  hardwareAccelerated={false}
  onRequestClose={[Function]}
  transparent={true}
  visible={false}
/>
`;

exports[`Test Modal isVisible true 1`] = `
<Modal
  animationType="slide"
  hardwareAccelerated={false}
  onRequestClose={[Function]}
  transparent={true}
  visible={true}
>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        undefined,
      ]
    }
    testID="RNE__Overlay__backdrop"
  />
  <RNCSafeAreaView
    pointerEvents="box-none"
    style={
      Object {
        "backgroundColor": "rgba(77, 77, 77, 0.9)",
        "flex": 1,
        "flexDirection": "column-reverse",
      }
    }
    theme={
      Object {
        "colors": Object {
          "background": "#ffffff",
          "black": "#242424",
          "disabled": "hsl(208, 8%, 90%)",
          "divider": "#bcbbc1",
          "error": "#ff190c",
          "grey0": "#393e42",
          "grey1": "#43484d",
          "grey2": "#5e6977",
          "grey3": "#86939e",
          "grey4": "#bdc6cf",
          "grey5": "#e1e8ee",
          "greyOutline": "#bbb",
          "platform": Object {
            "android": Object {
              "error": "#f44336",
              "grey": "rgba(0, 0, 0, 0.54)",
              "primary": "#2196f3",
              "searchBg": "#dcdce1",
              "secondary": "#9C27B0",
              "success": "#4caf50",
              "warning": "#ffeb3b",
            },
            "default": Object {
              "error": "#ff3b30",
              "grey": "#7d7d7d",
              "primary": "#007aff",
              "searchBg": "#dcdce1",
              "secondary": "#5856d6",
              "success": "#4cd964",
              "warning": "#ffcc00",
            },
            "ios": Object {
              "error": "#ff3b30",
              "grey": "#7d7d7d",
              "primary": "#007aff",
              "searchBg": "#dcdce1",
              "secondary": "#5856d6",
              "success": "#4cd964",
              "warning": "#ffcc00",
            },
            "web": Object {
              "error": "#ff190c",
              "grey": "#393e42",
              "primary": "#2089dc",
              "searchBg": "#303337",
              "secondary": "#ca71eb",
              "success": "#52c41a",
              "warning": "#faad14",
            },
          },
          "primary": "#2089dc",
          "searchBg": "#303337",
          "secondary": "#03dac4",
          "success": "#52c41a",
          "warning": "#faad14",
          "white": "#ffffff",
        },
        "spacing": Object {
          "lg": 12,
          "md": 8,
          "sm": 4,
          "xl": 24,
          "xs": 2,
        },
      }
    }
  >
    <View>
      <RCTScrollView>
        <View>
          <View
            style={
              Object {
                "alignItems": "center",
              }
            }
          />
        </View>
      </RCTScrollView>
    </View>
  </RNCSafeAreaView>
</Modal>
`;

exports[`Test Modal with ModalItems 1`] = `
<Modal
  animationType="slide"
  hardwareAccelerated={false}
  onRequestClose={[Function]}
  transparent={true}
  visible={true}
>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        undefined,
      ]
    }
    testID="RNE__Overlay__backdrop"
  />
  <RNCSafeAreaView
    pointerEvents="box-none"
    style={
      Object {
        "backgroundColor": "rgba(77, 77, 77, 0.9)",
        "flex": 1,
        "flexDirection": "column-reverse",
      }
    }
    theme={
      Object {
        "colors": Object {
          "background": "#ffffff",
          "black": "#242424",
          "disabled": "hsl(208, 8%, 90%)",
          "divider": "#bcbbc1",
          "error": "#ff190c",
          "grey0": "#393e42",
          "grey1": "#43484d",
          "grey2": "#5e6977",
          "grey3": "#86939e",
          "grey4": "#bdc6cf",
          "grey5": "#e1e8ee",
          "greyOutline": "#bbb",
          "platform": Object {
            "android": Object {
              "error": "#f44336",
              "grey": "rgba(0, 0, 0, 0.54)",
              "primary": "#2196f3",
              "searchBg": "#dcdce1",
              "secondary": "#9C27B0",
              "success": "#4caf50",
              "warning": "#ffeb3b",
            },
            "default": Object {
              "error": "#ff3b30",
              "grey": "#7d7d7d",
              "primary": "#007aff",
              "searchBg": "#dcdce1",
              "secondary": "#5856d6",
              "success": "#4cd964",
              "warning": "#ffcc00",
            },
            "ios": Object {
              "error": "#ff3b30",
              "grey": "#7d7d7d",
              "primary": "#007aff",
              "searchBg": "#dcdce1",
              "secondary": "#5856d6",
              "success": "#4cd964",
              "warning": "#ffcc00",
            },
            "web": Object {
              "error": "#ff190c",
              "grey": "#393e42",
              "primary": "#2089dc",
              "searchBg": "#303337",
              "secondary": "#ca71eb",
              "success": "#52c41a",
              "warning": "#faad14",
            },
          },
          "primary": "#2089dc",
          "searchBg": "#303337",
          "secondary": "#03dac4",
          "success": "#52c41a",
          "warning": "#faad14",
          "white": "#ffffff",
        },
        "spacing": Object {
          "lg": 12,
          "md": 8,
          "sm": 4,
          "xl": 24,
          "xs": 2,
        },
      }
    }
  >
    <View>
      <RCTScrollView>
        <View>
          <View
            style={
              Object {
                "alignItems": "center",
              }
            }
          >
            <View
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ffffff",
                  "borderBottomLeftRadius": 12,
                  "borderBottomRightRadius": 12,
                  "borderBottomWidth": 0.5,
                  "borderColor": "#bcbbc1",
                  "flexDirection": "row",
                  "padding": 14,
                  "width": "90%",
                }
              }
              testID="RNE__LISTITEM__padView"
            >
              <View
                style={
                  Array [
                    Object {
                      "alignItems": "flex-start",
                      "flex": 1,
                      "justifyContent": "center",
                    },
                    Object {
                      "alignItems": "center",
                    },
                  ]
                }
                theme={
                  Object {
                    "colors": Object {
                      "background": "#ffffff",
                      "black": "#242424",
                      "disabled": "hsl(208, 8%, 90%)",
                      "divider": "#bcbbc1",
                      "error": "#ff190c",
                      "grey0": "#393e42",
                      "grey1": "#43484d",
                      "grey2": "#5e6977",
                      "grey3": "#86939e",
                      "grey4": "#bdc6cf",
                      "grey5": "#e1e8ee",
                      "greyOutline": "#bbb",
                      "platform": Object {
                        "android": Object {
                          "error": "#f44336",
                          "grey": "rgba(0, 0, 0, 0.54)",
                          "primary": "#2196f3",
                          "searchBg": "#dcdce1",
                          "secondary": "#9C27B0",
                          "success": "#4caf50",
                          "warning": "#ffeb3b",
                        },
                        "default": Object {
                          "error": "#ff3b30",
                          "grey": "#7d7d7d",
                          "primary": "#007aff",
                          "searchBg": "#dcdce1",
                          "secondary": "#5856d6",
                          "success": "#4cd964",
                          "warning": "#ffcc00",
                        },
                        "ios": Object {
                          "error": "#ff3b30",
                          "grey": "#7d7d7d",
                          "primary": "#007aff",
                          "searchBg": "#dcdce1",
                          "secondary": "#5856d6",
                          "success": "#4cd964",
                          "warning": "#ffcc00",
                        },
                        "web": Object {
                          "error": "#ff190c",
                          "grey": "#393e42",
                          "primary": "#2089dc",
                          "searchBg": "#303337",
                          "secondary": "#ca71eb",
                          "success": "#52c41a",
                          "warning": "#faad14",
                        },
                      },
                      "primary": "#2089dc",
                      "searchBg": "#303337",
                      "secondary": "#03dac4",
                      "success": "#52c41a",
                      "warning": "#faad14",
                      "white": "#ffffff",
                    },
                    "spacing": Object {
                      "lg": 12,
                      "md": 8,
                      "sm": 4,
                      "xl": 24,
                      "xs": 2,
                    },
                  }
                }
              >
                <Text
                  accessibilityRole="text"
                  style={
                    Object {
                      "backgroundColor": "transparent",
                      "color": "#a72b1e",
                      "fontSize": 17,
                    }
                  }
                  testID="listItemTitle"
                >
                  
                </Text>
              </View>
            </View>
            <View
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ffffff",
                  "borderBottomLeftRadius": 12,
                  "borderBottomRightRadius": 12,
                  "borderBottomWidth": 0.5,
                  "borderColor": "#bcbbc1",
                  "flexDirection": "row",
                  "padding": 14,
                  "width": "90%",
                }
              }
              testID="RNE__LISTITEM__padView"
            >
              <View
                style={
                  Array [
                    Object {
                      "alignItems": "flex-start",
                      "flex": 1,
                      "justifyContent": "center",
                    },
                    Object {
                      "alignItems": "center",
                    },
                  ]
                }
                theme={
                  Object {
                    "colors": Object {
                      "background": "#ffffff",
                      "black": "#242424",
                      "disabled": "hsl(208, 8%, 90%)",
                      "divider": "#bcbbc1",
                      "error": "#ff190c",
                      "grey0": "#393e42",
                      "grey1": "#43484d",
                      "grey2": "#5e6977",
                      "grey3": "#86939e",
                      "grey4": "#bdc6cf",
                      "grey5": "#e1e8ee",
                      "greyOutline": "#bbb",
                      "platform": Object {
                        "android": Object {
                          "error": "#f44336",
                          "grey": "rgba(0, 0, 0, 0.54)",
                          "primary": "#2196f3",
                          "searchBg": "#dcdce1",
                          "secondary": "#9C27B0",
                          "success": "#4caf50",
                          "warning": "#ffeb3b",
                        },
                        "default": Object {
                          "error": "#ff3b30",
                          "grey": "#7d7d7d",
                          "primary": "#007aff",
                          "searchBg": "#dcdce1",
                          "secondary": "#5856d6",
                          "success": "#4cd964",
                          "warning": "#ffcc00",
                        },
                        "ios": Object {
                          "error": "#ff3b30",
                          "grey": "#7d7d7d",
                          "primary": "#007aff",
                          "searchBg": "#dcdce1",
                          "secondary": "#5856d6",
                          "success": "#4cd964",
                          "warning": "#ffcc00",
                        },
                        "web": Object {
                          "error": "#ff190c",
                          "grey": "#393e42",
                          "primary": "#2089dc",
                          "searchBg": "#303337",
                          "secondary": "#ca71eb",
                          "success": "#52c41a",
                          "warning": "#faad14",
                        },
                      },
                      "primary": "#2089dc",
                      "searchBg": "#303337",
                      "secondary": "#03dac4",
                      "success": "#52c41a",
                      "warning": "#faad14",
                      "white": "#ffffff",
                    },
                    "spacing": Object {
                      "lg": 12,
                      "md": 8,
                      "sm": 4,
                      "xl": 24,
                      "xs": 2,
                    },
                  }
                }
              >
                <Text
                  accessibilityRole="text"
                  style={
                    Object {
                      "backgroundColor": "transparent",
                      "color": "#a72b1e",
                      "fontSize": 17,
                    }
                  }
                  testID="listItemTitle"
                >
                  
                </Text>
              </View>
            </View>
            <View
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "transparent",
                  "borderBottomWidth": 0.5,
                  "borderColor": "#bcbbc1",
                  "flexDirection": "row",
                  "padding": 14,
                  "paddingBottom": 60,
                }
              }
              testID="RNE__LISTITEM__padView"
            >
              <View
                style={
                  Array [
                    Object {
                      "alignItems": "flex-start",
                      "flex": 1,
                      "justifyContent": "center",
                    },
                    Object {
                      "alignItems": "center",
                    },
                  ]
                }
                theme={
                  Object {
                    "colors": Object {
                      "background": "#ffffff",
                      "black": "#242424",
                      "disabled": "hsl(208, 8%, 90%)",
                      "divider": "#bcbbc1",
                      "error": "#ff190c",
                      "grey0": "#393e42",
                      "grey1": "#43484d",
                      "grey2": "#5e6977",
                      "grey3": "#86939e",
                      "grey4": "#bdc6cf",
                      "grey5": "#e1e8ee",
                      "greyOutline": "#bbb",
                      "platform": Object {
                        "android": Object {
                          "error": "#f44336",
                          "grey": "rgba(0, 0, 0, 0.54)",
                          "primary": "#2196f3",
                          "searchBg": "#dcdce1",
                          "secondary": "#9C27B0",
                          "success": "#4caf50",
                          "warning": "#ffeb3b",
                        },
                        "default": Object {
                          "error": "#ff3b30",
                          "grey": "#7d7d7d",
                          "primary": "#007aff",
                          "searchBg": "#dcdce1",
                          "secondary": "#5856d6",
                          "success": "#4cd964",
                          "warning": "#ffcc00",
                        },
                        "ios": Object {
                          "error": "#ff3b30",
                          "grey": "#7d7d7d",
                          "primary": "#007aff",
                          "searchBg": "#dcdce1",
                          "secondary": "#5856d6",
                          "success": "#4cd964",
                          "warning": "#ffcc00",
                        },
                        "web": Object {
                          "error": "#ff190c",
                          "grey": "#393e42",
                          "primary": "#2089dc",
                          "searchBg": "#303337",
                          "secondary": "#ca71eb",
                          "success": "#52c41a",
                          "warning": "#faad14",
                        },
                      },
                      "primary": "#2089dc",
                      "searchBg": "#303337",
                      "secondary": "#03dac4",
                      "success": "#52c41a",
                      "warning": "#faad14",
                      "white": "#ffffff",
                    },
                    "spacing": Object {
                      "lg": 12,
                      "md": 8,
                      "sm": 4,
                      "xl": 24,
                      "xs": 2,
                    },
                  }
                }
              >
                <Text
                  accessibilityRole="text"
                  style={
                    Object {
                      "backgroundColor": "transparent",
                      "color": "#fff",
                      "fontSize": 17,
                    }
                  }
                  testID="listItemTitle"
                >
                  
                </Text>
              </View>
            </View>
            <View
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ffffff",
                  "borderBottomWidth": 0.5,
                  "borderColor": "#bcbbc1",
                  "borderTopLeftRadius": 12,
                  "borderTopRightRadius": 12,
                  "flexDirection": "row",
                  "padding": 14,
                  "width": "90%",
                }
              }
              testID="RNE__LISTITEM__padView"
            >
              <View
                style={
                  Array [
                    Object {
                      "alignItems": "flex-start",
                      "flex": 1,
                      "justifyContent": "center",
                    },
                    Object {
                      "alignItems": "center",
                    },
                  ]
                }
                theme={
                  Object {
                    "colors": Object {
                      "background": "#ffffff",
                      "black": "#242424",
                      "disabled": "hsl(208, 8%, 90%)",
                      "divider": "#bcbbc1",
                      "error": "#ff190c",
                      "grey0": "#393e42",
                      "grey1": "#43484d",
                      "grey2": "#5e6977",
                      "grey3": "#86939e",
                      "grey4": "#bdc6cf",
                      "grey5": "#e1e8ee",
                      "greyOutline": "#bbb",
                      "platform": Object {
                        "android": Object {
                          "error": "#f44336",
                          "grey": "rgba(0, 0, 0, 0.54)",
                          "primary": "#2196f3",
                          "searchBg": "#dcdce1",
                          "secondary": "#9C27B0",
                          "success": "#4caf50",
                          "warning": "#ffeb3b",
                        },
                        "default": Object {
                          "error": "#ff3b30",
                          "grey": "#7d7d7d",
                          "primary": "#007aff",
                          "searchBg": "#dcdce1",
                          "secondary": "#5856d6",
                          "success": "#4cd964",
                          "warning": "#ffcc00",
                        },
                        "ios": Object {
                          "error": "#ff3b30",
                          "grey": "#7d7d7d",
                          "primary": "#007aff",
                          "searchBg": "#dcdce1",
                          "secondary": "#5856d6",
                          "success": "#4cd964",
                          "warning": "#ffcc00",
                        },
                        "web": Object {
                          "error": "#ff190c",
                          "grey": "#393e42",
                          "primary": "#2089dc",
                          "searchBg": "#303337",
                          "secondary": "#ca71eb",
                          "success": "#52c41a",
                          "warning": "#faad14",
                        },
                      },
                      "primary": "#2089dc",
                      "searchBg": "#303337",
                      "secondary": "#03dac4",
                      "success": "#52c41a",
                      "warning": "#faad14",
                      "white": "#ffffff",
                    },
                    "spacing": Object {
                      "lg": 12,
                      "md": 8,
                      "sm": 4,
                      "xl": 24,
                      "xs": 2,
                    },
                  }
                }
              >
                <Text
                  accessibilityRole="text"
                  style={
                    Object {
                      "backgroundColor": "transparent",
                      "color": "#242424",
                      "fontSize": 17,
                    }
                  }
                  testID="listItemTitle"
                >
                  
                </Text>
              </View>
            </View>
            <View
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ffffff",
                  "borderBottomWidth": 0.5,
                  "borderColor": "#bcbbc1",
                  "borderTopLeftRadius": 12,
                  "borderTopRightRadius": 12,
                  "flexDirection": "row",
                  "padding": 14,
                  "width": "90%",
                }
              }
              testID="RNE__LISTITEM__padView"
            >
              <View
                style={
                  Array [
                    Object {
                      "alignItems": "flex-start",
                      "flex": 1,
                      "justifyContent": "center",
                    },
                    Object {
                      "alignItems": "center",
                    },
                  ]
                }
                theme={
                  Object {
                    "colors": Object {
                      "background": "#ffffff",
                      "black": "#242424",
                      "disabled": "hsl(208, 8%, 90%)",
                      "divider": "#bcbbc1",
                      "error": "#ff190c",
                      "grey0": "#393e42",
                      "grey1": "#43484d",
                      "grey2": "#5e6977",
                      "grey3": "#86939e",
                      "grey4": "#bdc6cf",
                      "grey5": "#e1e8ee",
                      "greyOutline": "#bbb",
                      "platform": Object {
                        "android": Object {
                          "error": "#f44336",
                          "grey": "rgba(0, 0, 0, 0.54)",
                          "primary": "#2196f3",
                          "searchBg": "#dcdce1",
                          "secondary": "#9C27B0",
                          "success": "#4caf50",
                          "warning": "#ffeb3b",
                        },
                        "default": Object {
                          "error": "#ff3b30",
                          "grey": "#7d7d7d",
                          "primary": "#007aff",
                          "searchBg": "#dcdce1",
                          "secondary": "#5856d6",
                          "success": "#4cd964",
                          "warning": "#ffcc00",
                        },
                        "ios": Object {
                          "error": "#ff3b30",
                          "grey": "#7d7d7d",
                          "primary": "#007aff",
                          "searchBg": "#dcdce1",
                          "secondary": "#5856d6",
                          "success": "#4cd964",
                          "warning": "#ffcc00",
                        },
                        "web": Object {
                          "error": "#ff190c",
                          "grey": "#393e42",
                          "primary": "#2089dc",
                          "searchBg": "#303337",
                          "secondary": "#ca71eb",
                          "success": "#52c41a",
                          "warning": "#faad14",
                        },
                      },
                      "primary": "#2089dc",
                      "searchBg": "#303337",
                      "secondary": "#03dac4",
                      "success": "#52c41a",
                      "warning": "#faad14",
                      "white": "#ffffff",
                    },
                    "spacing": Object {
                      "lg": 12,
                      "md": 8,
                      "sm": 4,
                      "xl": 24,
                      "xs": 2,
                    },
                  }
                }
              >
                <Text
                  accessibilityRole="text"
                  style={
                    Object {
                      "backgroundColor": "transparent",
                      "color": "#fff",
                      "fontSize": 17,
                    }
                  }
                  testID="listItemTitle"
                >
                  
                </Text>
              </View>
            </View>
          </View>
        </View>
      </RCTScrollView>
    </View>
  </RNCSafeAreaView>
</Modal>
`;
